addons/viewport/register.js

```diff
@@ -0,0 +1,3 @@
+// NOTE: loading addons using this file is deprecated!
+// please use manager.js and preview.js files instead
+require('./manager');
```

Comment:
@ndelangen @mnmtanish tangential to this PR but do you know when `register` was deprecated? This is the first time I've seen it. And isn't it strange that all of our existing addons use a deprecated format (AFAIK)? Seems like something we should figure out?

Reply:
It's also weird that we actually recommend importing this file in addon's documentation